### ISSUE TITLE
RESIDENT EVIL 6 [BLUS30855] Mosaic appears where there is a light source since build number 0.0.9-9929

### ISSUE THREAD

**Issue by zombie167** [RPCS3/rpcs3#7818]
Mosaic appears where there is a light source since the PR:vk: Lazy evaluate renderpass scope #7784 is being merged into the master build.Previous build 0.0.9-9928 worked fine

0.0.9-9928
![9928](https://user-images.githubusercontent.com/22722505/77228601-71c8c800-6bc3-11ea-9a2e-13600de9a092.jpg)

0.0.9-9929
![9929](https://user-images.githubusercontent.com/22722505/77228613-7b523000-6bc3-11ea-9af4-36c7df54f346.jpg)

[log.zip](https://github.com/RPCS3/rpcs3/files/4363115/log.zip)

**Comment by kd-11** [comment]
Blocked by https://github.com/RPCS3/rpcs3/issues/6639

**Comment by kd-11** [comment]
The cause of this bug and the related https://github.com/RPCS3/rpcs3/issues/7830 has been identified, but would require extensive rewrites to implement correctly. I'll think up a workaround in the meantime to get the desired effect without hurting performance too much.

**Comment by kd-11** [comment]
Test with https://github.com/RPCS3/rpcs3/pull/8563

**Comment by zombie167** [comment]
Thank you very much!The problem has been solved
